### DISCUSSION TITLE
Exported xcode project from Unity appears black screen on iOS Simulator

### DISCUSSION THREAD

**Discussion by steven90311** [community/community#63643] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Hello.

I am working with iOS game from Unity.
I exported iOS project from Unity for simulator. But there is only black screen on iOS simulator.
Environment is macOS 13 vm and xcode version is 14.
Please help me...

**Comment by eddytalvala** [comment, marked as answer]
Your vm doesn't support metal-compatible virtual hardware.
please check this post.
https://stackoverflow.com/questions/73721978/unity3d-build-for-ios-simulator-in-xcode-displays-a-black-screen-and-an-error-in#:~:text=your%20VM%20doesn%27t%20provide%20metal%2Dcompatible%20virtual%20hardware.

**Reply by steven90311** [reply]
Thank you for your answer
